lib/pure/parseutils.nim

```diff
@@ -71,21 +71,20 @@ proc parseBin*[T: SomeInteger](s: string, number: var T, start = 0,
   ## are parsed starting from the ``start`` position.
   ##
   ## It does not check for overflow. If the value represented by the string is
-  ## too big to fit into ``number``, only the value of last fitting characters
-  ## will be stored in ``number`` without producing an error.
+  ## too big to fit into ``number``, 0 will be returned indicating an error.
   runnableExamples:
     var num: int
     doAssert parseBin("0100_1110_0110_1001_1110_1101", num) == 29
     doAssert num == 5138925
     doAssert parseBin("3", num) == 0
-    var num8: int8
-    doAssert parseBin("0b_0100_1110_0110_1001_1110_1101", num8) == 32
-    doAssert num8 == 0b1110_1101'i8
-    doAssert parseBin("0b_0100_1110_0110_1001_1110_1101", num8, 3, 9) == 9
-    doAssert num8 == 0b0100_1110'i8
-    var num8u: uint8
-    doAssert parseBin("0b_0100_1110_0110_1001_1110_1101", num8u) == 32
-    doAssert num8u == 237
+    # var num8: int8
+    # doAssert parseBin("0b_0100_1110_0110_1001_1110_1101", num8) == 32
+    # doAssert num8 == 0b1110_1101'i8
+    # doAssert parseBin("0b_0100_1110_0110_1001_1110_1101", num8, 3, 9) == 9
+    # doAssert num8 == 0b0100_1110'i8
+    # var num8u: uint8
+    # doAssert parseBin("0b_0100_1110_0110_1001_1110_1101", num8u) == 32
+    # doAssert num8u == 237
```

Comment:
They are all invalid now

Reply:
but this shouldn't be in docs, and `when false` is preferable to comments; move the block to a `when false;` outside of the runnableExamples

Reply:
yeah, I comment it for discussing.

Reply:
maybe we need to dig in git log to figure out why it was introduced like that (but it does look suspicious)

Reply:
fixed